optuna/storages/rdb/storage.py

```diff
@@ -602,3 +594,138 @@ def __del__(self):
 
         if hasattr(self, 'scoped_session'):
             self.remove_session()
+
```

Comment:
Since `RDBStorage` is getting fat, how about splitting version management logic to another class?

Reply:
Sounds good! Thank you for your suggestion.

Reply:
I added `_VersionManager` class (see: https://github.com/pfnet/optuna/pull/336/commits/187ff9cbb00a07c789897a717c65c8bbf3b21164).